CHANGELOG.md

```diff
@@ -8,12 +8,39 @@ Please keep this up to date, following the [instructions](#instructions) below.
 
 ## [Unreleased](https://github.com/opendp/opendp/compare/stable...HEAD)
 
+### Added
+- FFI and Python interfaces for creating and accessing Domains, Metrics, and Measures ([#637](https://github.com/opendp/opendp/pull/637))
+- Queryables and supporting infrastructure for interactive Measurements ([#618](https://github.com/opendp/opendp/pull/618)), ([#675](https://github.com/opendp/opendp/pull/675))
+- Constructor for sequential compostion interactive Measurement ([#674](https://github.com/opendp/opendp/pull/674))
+- Checks for compatibility between pairings of Domains and Metrics/Measures ([#604](https://github.com/opendp/opendp/pull/604))
+- Python `opendp.extrinsics` module for code contributions and proofs outside of Rust ([#693](https://github.com/opendp/opendp/pull/693))
+- Docs: [First Look at DP](https://docs.opendp.org/en/stable/user/first-look-at-DP.html) notebook ([#666](https://github.com/opendp/opendp/pull/666))
+
+### Changed
+- Incorporated Domain instances into some constructor signatures ([#650](https://github.com/opendp/opendp/pull/650))
+- Simplified postprocessors to Function (from previous full Transformation) ([#648](https://github.com/opendp/opendp/pull/648))
+- Moved some Domain logic from type-inherent constraints to runtime checks of more general types ([#645](https://github.com/opendp/opendp/pull/645)), ([#696](https://github.com/opendp/opendp/pull/696))
+  - Remove SizedDomain in favor of a runtime size descriptor on VectorDomain
+  - Remove BoundedDomain in favor of a runtime bounds descriptor on AtomDomain
+  - Remove InherentNullDomain in favor of a runtime nullity descriptor on AtomDomain
+- Docs: Restructured and improved the [User Guide](https://docs.opendp.org/en/stable/user/index.html) ([#639](https://github.com/opendp/opendp/pull/639))
+- Docs: Renamed the Developer Guide to [Contributor Guide](https://docs.opendp.org/en/stable/contributor/index.html) ([#639](https://github.com/opendp/opendp/pull/639))
+
+### Deprecated
+- AllDomain in the Python bindings, with a warning to switch to AtomDomain ([#645](https://github.com/opendp/opendp/pull/645))
+
+### Removed
+- The `output_domain` field of Measurement struct ([#647](https://github.com/opendp/opendp/pull/647))
```

Comment:
`make_default_user_transformation`, `make_default_user_measurement`, `make_default_postprocessor` have been removed. Might put this into the "changed" section because we have better alternatives now.